Napzak-iOS/Napzak-iOS/Global/Style/CustomToggleStyle.swift

```diff
@@ -26,28 +29,83 @@ struct CustomToggleStyle: ToggleStyle {
                     .animation(.easeInOut(duration: 0.2), value: configuration.isOn)
             }
             .onTapGesture {
-                configuration.isOn.toggle()
+                if isEnabled {
+                    configuration.isOn.toggle()
+                }
+            }
+        }
+    }
+    
+    private func backgroundColor(for configuration: Configuration) -> Color {
+        if isEnabled {
+            if configuration.isOn {
+                return Color.napzakPrimary(.purple500)
+            } else {
+                return Color.gray.opacity(0.4)
+            }
+        } else {
+            if configuration.isOn {
+                return Color.napzakPrimary(.purple200)
+            } else {
+                return Color.gray.opacity(0.4)
             }
         }
```

Comment:
P2

피그마에 비활성 토글 색상 `gray100`으로 되어있어서 적용해주시면 좋을 것 같습니다!
또한 삼항 연산자 사용하면 좀 더 간결하고 가독성있게 보완 가능한 것 같아서 달아드려요!

```suggestion
        if configuration.isOn {
            return Color.napzakPrimary(isEnabled ? .purple500 : .purple200)
        } else {
            return Color.napzakGrayScale(.gray100)
        }
```